packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,256 @@ def _clean_html(self, html: str) -> str:
+                            "No cursor found in next URL, ending pagination"
+                        )
                         break
+                    logger.debug(f"Found next cursor: {cursor}")
+                except (ValueError, KeyError, AttributeError) as e:
+                    logger.error(f"Failed to parse next URL: {e!s}")
+                    break
+
+            except (requests.exceptions.RequestException, ValueError) as e:
+                logger.error(
+                    f"Failed to fetch content from space {self.config.space_key}: {e!s}"
+                )
```

Ruff (0.15.20)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)